scripts/build_info.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+HEAD=$(git rev-parse --verify HEAD --short)
+DATE=$(date +%Y-%m-%d)
+TIME=$(date +%H:%M:%S)
```

Comment:
This is date and time from the system. I would use the information from the latest commit.

Reply:
Good point, I was thinking about it like the "build time", but it's maybe more interesting the "commit time".